website/views/options/social/chat-message.jade

```diff
@@ -1,6 +1,6 @@
 mixin chatMessages(inbox)
   ul.list-unstyled.tavern-chat
-    - var ngRepeat = inbox ? 'message in user.inbox.messages | toArray:true | orderBy:"sort":true' : 'message in group.chat track by message.id'
+    - var ngRepeat = inbox ? 'message in user.inbox.messages track by $index | toArray:true | orderBy:"sort":true' : 'message in group.chat track by message.id'
```

Comment:
I think it's because I put the track by filter in the wrong part of the statement. Cleaning this up, then I would appreciate another look :)